components/search/SearchForm.vue

```diff
@@ -35,14 +35,24 @@
       };
     },
     computed: {
+      onSearchPage() {
```

Comment:
onSearchPage suggests this is only used on the /search page. What about onSearchablePage, to also acknowledge the entity pages?

Reply:
Done.